docs/Guides/behaviour/ps_routing.rst

```diff
@@ -0,0 +1,90 @@
+If the arrival distribution is Poisson with rate 8, and required service times are exponentially distributed with parameter 10, then our network is::
+
+    >>> import ciw
+    >>> N = ciw.create_network(
+    ...     arrival_distributions=[ciw.dists.Exponential(8),
+    ...                            ciw.dists.NoArrivals(),
+    ...                            ciw.dists.NoArrivals(),
+    ...                            ciw.dists.NoArrivals()],
+    ...     service_distributions=[ciw.dists.Deterministic(0),
+    ...                            ciw.dists.Exponential(10),
+    ...                            ciw.dists.Exponential(10),
+    ...                            ciw.dists.Exponential(10)],
+    ...     number_of_servers=[float('inf'),
+    ...                        float('inf'),
+    ...                        float('inf'),
+    ...                        float('inf')],
+    ...     routing=[[0, 0, 0, 0],
+    ...              [0, 0, 0, 0],
+    ...              [0, 0, 0, 0],
+    ...              [0, 0, 0, 0]]
+    ... )
+
+For each of the three parallel processor sharing nodes, we can use the :code:`ciw.PSNode` class.
+However, we now need a custom node class for the initial dummy node, to take care of the routing decisions.
+We'll call this class :code:`RoutingDecision`::
+
+    >>> class RoutingDecision(ciw.Node):
+    ...     def next_node(self, ind):
+    ...         """
+    ...         Finds the next node by looking at nodes 2, 3, and 4,
+    ...         seeing how busy they are, and routing to the least busy.
+    ...         """
+    ...         busyness = {n: self.simulation.nodes[n].number_of_individuals for n in [2, 3, 4]}
+    ...         chosen_n = sorted(busyness.keys(), key=lambda x: busyness[x])[0]
+    ...         return self.simulation.nodes[chosen_n]
+
+Now let's build a simulation object, where the first node uses our custom :code:`RoutingDecision` class, and the others use the built-in :code:`ciw.PSNode` class. We'll also add a state tracker for analysis::
+
+    >>> ciw.seed(0)
+    >>> Q = ciw.Simulation(
+    ...     N, tracker=ciw.trackers.SystemPopulation(),
+    ...     node_class=[RoutingDecision, ciw.PSNode, ciw.PSNode, ciw.PSNode])
+
+We'll run this for 100 time units::
+
+    >>> Q.simulate_until_max_time(100)
+
+We can look at the state probabilities, that is, the proportion of time the system spent in each state, where a state represents the number of customers present in the system:
+
+    >>> Q.statetracker.state_probabilities(observation_period=(10, 90)) # doctest:+SKIP
+    {0: 0.425095024227593,
+     1: 0.35989517302304014,
+     2: 0.14629711075255158,
+     3: 0.054182634504608064,
+     4: 0.01124224242623659,
+     5: 0.002061285633093934,
+     6: 0.0012265294328765105}
+
+In [XL09]_ theoretical results are given for these probabilities by:
```

Comment:
@geraintpalmer the system considered by Dr. Li is slightly different.
On her model, if the number of customers is >R, then they are processed in a G/G/1-PS sharing with a single server of service rate equal to R·\mu.

Thus, I would not compare against this formula.
If they match, it may be because of the \lambda and  \mu election.

In case it always matches, then we have here an interesting mathematical result stating that Dr. Li system has same probabilities as a G/G/R-PS system.
I think that requires a mathematical proof that we could present in IEEE COMLET, for example.
Let me know if you'd be interesting on further investigating it.

However, I may be wrong on interpreting Dr. Li thesis, you can check it and tell my if you agree on my thoughts.